tests/test_deepgrow_transforms.py

```diff
@@ -312,7 +312,7 @@
 ]
 
 RESTORE_LABEL_TEST_CASE_1 = [
-    {"keys": ["pred"], "ref_image": "image", "mode": "nearest"},
+    {"keys": ["pred"], "ref_image": "image", "mode": "nearest", "restore_resize": True, "restore_crop": True, "restore_spacing": True, "restore_slicing": True},
```

Comment:
I don't think changing this dictionary affects anything, it just adds the default values for these parameters. The addition here I think can be omitted.
```suggestion
    {"keys": ["pred"], "ref_image": "image", "mode": "nearest"},
```